### PR TITLE
Update fn_weaponShopFilter.sqf

### PR DESCRIPTION
**Pull request by fresqo**
gives ability to restrict weapons and accessories by level individually, by using Config_weapons.hpp like: 
{ "srifle_DMR_03_woodland_F", "", 20500, -1,{"life_coplevel", "SCALAR", 5 }},
OR
 { "optic_Arco", "", 2500, -1, {"life_coplevel","SCALAR",4} },

### CODE REVIEW

File: Altis_Life.Altis/core/shops/fn_weaponShopFilter.sqf [hunk 1]
```diff
@@ -35,15 +35,39 @@ if((GVAR_UINS ["Weapon_Magazine",0]) == 1 OR (GVAR_UINS ["Weapon_Accessories",0]
...
+                    default {false};
+                    };
+                    if (_bool) then {
+                            _itemInfo = [SEL(_x,0)] call life_fnc_fetchCfgDetails;
+                            _itemList lbAdd format["%1",if(!(EQUAL(SEL(_x,1),""))) then {SEL(_x,1)} else {_itemInfo select 1}];
+                            _itemList lbSetData[(lbSize _itemList)-1,_itemInfo select 0];
+                            _itemList lbSetPicture[(lbSize _itemList)-1,_itemInfo select 2];
+                            _itemList lbSetValue[(lbSize _itemList)-1,SEL(_x,2)];
+                        };
+                };} else {
```
Comment (sargken) [inline Altis_Life.Altis/core/shops/fn_weaponShopFilter.sqf]: That looks like an error or not properly indented. - sent from lg

File: Altis_Life.Altis/core/shops/fn_weaponShopFilter.sqf [hunk 2]
```diff
@@ -55,14 +79,39 @@ if((GVAR_UINS ["Weapon_Magazine",0]) == 1 OR (GVAR_UINS ["Weapon_Accessories",0]
...
```
Comment (sargken) [inline Altis_Life.Altis/core/shops/fn_weaponShopFilter.sqf]: Same as above

**Comment by fresqo** [comment]
works fine on our server, but yeah indenting prob not right sry, pasting code in didn't paste the same :/

**Comment by Jawshy** [comment]
> gives ability to restrict weapons and accessories by level individually...

But not magazines individually or by licence.

**Comment by fresqo** [comment]
we did discuss this in other chat i don't think anyone really needs to restrict a mag

**Comment by Heavybob** [comment]
No point to restrict the mag. 
Curious, is it not going to display the weapon if the lvl is higher?

**Comment by tkcjesse** [comment]
I completely disagree. On my server only certain ranks had 3GL smoke nade access.  And certain ranks got an additional color. Certain ranks only had lethal magazines as well. I think magazines should be done as well.

**Comment by fresqo** [comment]
i guess i can do another push after this one for mags, it wont hurt having the capability. But @danielstuart14 are you working on this general feature also?. don't wanna be wasting my time.

**Comment by Liam0** [comment]
Please don't just throw it in there with a shovel -.-

**Comment by tkcjesse** [comment]
Lol'd.

**Comment by Heavybob** [comment]
@tkcjesse No point to restrict mags since the weapon won't be available to even get the ammo for it. I can only think that you would want to restrict the use of tracer or higher capacity mags.

**Comment by Heavybob** [comment]
BUMP

**Comment by Heavybob** [comment]
bump

**Comment by Heavybob** [comment]
bump

**Comment by Liam0** [comment]
@fresqo hasn't put any changes in for 6 days so i'll just close it for now. Feel free to reopen it as soon as you feel its ready.

**Pull request closed without merging**